### ISSUE TITLE
Create a "Community Overview" dashboard for identifying committer candidates

### ISSUE THREAD

**Issue by hezyin** [apache/incubator-devlake#1995]
## Overview

As DevLake's awesome community continues to grow, we would like to keep track of the community contributions so that we can identify community members to promote to committers over time. This is a natural DevLake use case and I propose a "Community Overview" dashboard for this purpose.

This dashboard will focus on contributions in the format of pull requests, issues, and comments on either of those. Other types of contributions need to be tracked elsewhere.

## Metrics & Panels

1. Number of Unique PR Authors (and the trend)
2. Number of Unique Issue Authors (and the trend)
3. (Optional) Number of Unique Comment Authors (and the trend)
4. New PR Authors [Selected Time Range] (This panel is a table showing a list of new PR authors with their PR counts)
5. New Issue Authors [Selected Time Range] (This panel is a table showing a list of new Issue authors with their Issue counts)
6. Committers (A list of current committers, not sure where we can get this data though, from Apache GitHub org or Apache API?)
7. Committer Candidates (A list of users with the following info: when they join the community (first PR / Issue date), PR counts, Issue counts, comment counts, and etc.)

**Comment by Startrekzky** [comment]
I'll take it. It's blocked by #2122 as I can't filter community users by Github users' org info.
